Remove javascript file install, and also removes
the need of the EM++ EMCC compiler, not in the repo.
Also add DESTDIR to fix packaging install. 
DESTDIR fixes submitted upstream:
PR: https://github.com/google/jsonnet/pull/740
---
diff --git Makefile Makefile
index 474655c..c533125 100644
--- Makefile
+++ Makefile
@@ -82,10 +82,6 @@ LIBS = \
 	libjsonnet++.so.$(VERSION) \
 
 ALL = \
-	libjsonnet_test_snippet \
-	libjsonnet_test_file \
-	libjsonnet.js \
-	doc/js/libjsonnet.js \
 	$(BINS) \
 	$(LIBS) \
 	$(LIB_OBJ)
@@ -187,12 +183,6 @@ JS_EXPORTED_FUNCTIONS = 'EXPORTED_FUNCTIONS=["_jsonnet_make", "_jsonnet_evaluate
 JS_RUNTIME_METHODS = 'EXTRA_EXPORTED_RUNTIME_METHODS=["cwrap", "_free", "getValue", "lengthBytesUTF8", "_malloc", "UTF8ToString", "setValue", "stringToUTF8", "addFunction"]'
 
 
-libjsonnet.js: $(LIB_SRC) $(ALL_HEADERS)
-	$(EMCXX) -s WASM=0 -s $(JS_EXPORTED_FUNCTIONS) -s $(JS_RUNTIME_METHODS) $(EMCXXFLAGS) $(LDFLAGS) $(LIB_SRC) -o $@
-
-# Copy javascript build to doc directory
-doc/js/libjsonnet.js: libjsonnet.js
-	$(CP) $^ $@
 
 # Tests for C binding.
 LIBJSONNET_TEST_SNIPPET_SRCS = \
diff --git Makefile Makefile
index c533125..56fb1e4 100644
--- Makefile
+++ Makefile
@@ -122,12 +122,12 @@ endif
 default: jsonnet jsonnetfmt
 
 install: bins libs
-	mkdir -p $(PREFIX)/bin
-	cp $(BINS) $(PREFIX)/bin/
-	mkdir -p $(PREFIX)/lib
-	cp $(LIBS) $(PREFIX)/lib/
-	mkdir -p $(PREFIX)/include
-	cp $(INCS) $(PREFIX)/include/
+	mkdir -p $(DESTDIR)$(PREFIX)/bin
+	cp $(BINS) $(DESTDIR)$(PREFIX)/bin/
+	mkdir -p $(DESTDIR)$(PREFIX)/lib
+	cp $(LIBS) $(DESTDIR)$(PREFIX)/lib/
+	mkdir -p $(DESTDIR)$(PREFIX)/include
+	cp $(INCS) $(DESTDIR)$(PREFIX)/include/
 
 all: $(ALL)
 
